packages/features/bookings/components/FiltersContainer.tsx

```diff
@@ -92,51 +82,40 @@ const PeopleFilter = () => {
 };
 
 export function FiltersContainer() {
-  const { t } = useLocale();
+  const isFiltersVisible = useBookingMultiFilterStore((state) => state.isFiltersVisible, shallow);
 
-  const [addFilterOptions, toggleOption, isFilterActive] = useBookingMultiFilterStore((state) => [
-    state.addFilterOptions,
-    state.toggleOption,
-    state.isFilterActive,
-    shallow,
-  ]);
+  return isFiltersVisible ? (
+    <div className="no-scrollbar flex w-full space-x-2 overflow-x-scroll rtl:space-x-reverse">
+      <PeopleFilter />
+      <EventTypeFilter />
+      <TeamsFilter />
+    </div>
+  ) : null;
+}
 
-  const isPeopleFilterActive = isFilterActive("people");
-  const isEventTypeFilterActive = isFilterActive("event_type");
+export function FilterToggleCta() {
+  const {
+    data: { teamIds, userIds, eventTypeIds },
+  } = useFilterQuery();
+  const { t } = useLocale();
+  const toggleFiltersVisibility = useBookingMultiFilterStore(
+    (state) => state.toggleFiltersVisibility,
+    shallow
+  );
 
   return (
-    <div className="flex w-full space-x-2 rtl:space-x-reverse">
-      {addFilterOptions.length > 0 && (
-        <Dropdown>
-          <DropdownMenuTrigger asChild>
-            <div className="hover:border-emphasis border-default text-default hover:text-emphasis mb-4 flex h-9 max-h-72 items-center justify-between whitespace-nowrap rounded-md border px-3 py-2 text-sm hover:cursor-pointer focus:border-neutral-300 focus:outline-none focus:ring-2 focus:ring-neutral-800 focus:ring-offset-1">
-              <Plus className="mr-2 h-4 w-4" />
-              <Tooltip content={t("add_filter")}>
-                <div>{t("add_filter")}</div>
-              </Tooltip>
-            </div>
-          </DropdownMenuTrigger>
-          <DropdownMenuContent className="w-56">
-            {addFilterOptions?.map((option) => (
-              <DropdownMenuItem key={option.label}>
-                <DropdownItem
-                  type="button"
-                  StartIcon={option.StartIcon}
-                  onClick={() => {
-                    toggleOption(option);
-                  }}>
-                  {t(option.label)}
-                </DropdownItem>
-              </DropdownMenuItem>
-            ))}
-          </DropdownMenuContent>
-        </Dropdown>
+    <button
+      onClick={toggleFiltersVisibility}
+      className="hover:border-emphasis border-default text-default hover:text-emphasis mb-4 flex h-9 max-h-72 items-center justify-between whitespace-nowrap rounded-md border px-3 py-2 text-sm hover:cursor-pointer focus:border-neutral-300 focus:outline-none focus:ring-2 focus:ring-neutral-800 focus:ring-offset-1">
```

Comment:
We already have Button component in @calcom/ui. we can use   Button
with color="secondary" as prop

Reply:
I have made this change.